src/components/composite/grid/grid-section.tsx

```
@@ -4,23 +4,50 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { Box, Grid } from '@mui/material';
-import { FormattedMessage } from 'react-intl';
-import type { SxStyle } from '../../../utils/styles';
+import {Box, Grid, Tooltip} from '@mui/material';
+import {FormattedMessage, useIntl} from 'react-intl';
+import type {SxStyle} from '../../../utils';
+import {mergeSx} from "../../../utils";
+import {InfoOutlined} from "@mui/icons-material";
```

ESLint

[error] `@mui/icons-material` import should occur before type import of `../../../utils`

(import-x/order)